virttest/utils_test/__init__.py

```diff
@@ -2276,6 +2276,8 @@ def install(self):
         status, output = self.cmd_status_output(self.make_cmds,
                                                 timeout=self.stress_shell_timeout)
         if status != 0:
+            config_log = self.cmd_output_safe('cd %s;cat config.log' % install_path)
```

Comment:
It is not good to hard code the  log file name 'config.log'  here.
Suggestion:
1. Check the file's existence before reading it, Or 
2. 'cat *.log' if you want all the logs.

Reply:
This is the default log file, I don't think it needs check, especially when build fails.

Reply:
I recommend test of file existence as there can be different stress tools configured by user and I am not sure config.log would be present for all.